README.md

```diff
@@ -1,2 +1,58 @@
+- individuals zero prior IT experience  
+- individuals pursuing an IT degree  
+- individuals seek a career shift into IT  
+- mid and senior level IT professions trying to enter management  
+
+## Description  
+
+Participants will work under the guidance of a coach to complete open-source initiatives for inDevelopme's, with tasks being specific to the individual's selected coaching track (e.g. Programming, DevOps, Content Strategy). When necessary, participants shall be granted access to software licenses, credentails, and information to complete tasks.
+
+## Tracks 
+
+Participants can choose to receive guidance from 1 of 3 GSD-IT categories:
+- Programming
+- DevOps
+- Content Strategy
+
+### Programming
+
+This GSD-IT track focuses heavily on programming concepts and best practicing. GIT, Composer, Jetbrain, Laravel (PHP), and Flask (Python) represent some of the products and technologies used within this track. 
+
+***Programming: General Tasks***  
+
+Participants will work through a series of tasks meticulously designed to foster growth in programming languages (e.g. Python, Java, C++, etc). They will tackle debugging scenarios, conditional statements, iterative programming,practice user input and output handling, and build basic user interfaces. Other tasks will include self-exploration in data structures, algorithms, version control, etc to participant on small projects designed to allow application of newly acquired knowledge and skills.  
+
+***Programming: Benefits to Participant***  
+
+Consistent engagement in programming will help partcipants build a solid foundation in programming, enabling them to take on more complex challenges. Participants will also develop a portfolio of skills and work experience for the world to see!
+
+### DevOps  
+
+This GSD-IT track focuses heavily on devops concepts and best practices using AWS. Docker, Jenkins, Terraform, and Amazon Web Service (AWS) are just some of the products and technologies used within this track.
+
+***DevOps: General Tasks***  
+
+Participants will apply fundamental concepts and utilize essentials tools such as Jenkins, Docker, and AWS. The will learn to use and create IaaS, PaaS, and SaaS products and services. Participants will explore concepts such as Infrastructure as Code (IaC) & Continous Integration / Continuous Deployment (CI/CD) to complete tasks for inDevelopme's open-source libraries. A participant may be asked to learn about containerizaiton or container orchestration tools -- and then be asked to demonstration competency through an implementation. This has a heavy focus in sucrity best practices, collaboration, application architecture, and documentation. 
+  
+***DevOps: Benefits to Participant***  
+
+Consistent engagement in the devops track will help partcipants build a solid foundation in cloud architecture. Participants will develop skill that help solve high availability, reliability, and scalability issues within an cloud application. 
```

Comment:
- should be DevOps
- add an s to "skill"
- change "an" to "a" (a cloud application"